src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
```

Comment:
I've suggest change name. This will lead to `use App\Lib\Mail\Mail;` which not clear.